tests/lang/eval-okay-replacestrings.nix

```diff
@@ -8,4 +8,5 @@ with builtins;
   (replaceStrings [""] ["X"] "abc")
   (replaceStrings [""] ["X"] "")
   (replaceStrings ["-"] ["_"] "a-b")
+  (replaceStrings ["abc"] [(throw "unreachable")] "foobar")
```

Comment:
For completeness, you could also check that non-strictness is at the list _item_ level, for example:

```
  (replaceStrings ["oo" "XX"] ["u" (throw "unreachable")] "foobar")
```

Reply:
Noted, checking at the list item level is definitely more robust.